### DISCUSSION TITLE
Specter on one computer and Bitcoin Core on a separate RaspberryPi

### DISCUSSION THREAD

**Discussion by jmdeacon** [cryptoadvance/specter-desktop#857] [Nodeboxes]
Is it possible to have Specter installed on my main computer and have it connect to a full node on my RaspberryPi?

They are on the same local network.

**Comment by ben-kaufman** [comment]
Yes, just from Specter settings Bitcoin Core tab, turn off auto detect, and enter your RPC username and password, and put the node's local IP in the host field.

**Comment by jonathancross** [comment]
You can also us an ssh tunnel to make it look like the bitcoin core node is localhost like so:
```
ssh -nNT -L 8332:localhost:8332 user@your_RaspberryPi_node
```

Then Specter will just find it automatically.

**Comment by hollisp01** [comment]
If I put desktop Specter on laptop and then connect to RaspberryPi, do I still connect the cold wallet to the Pi or the Laptop for adding to Specter

**Reply by ben-kaufman** [reply]
The laptop

**Comment by jdelgadoesteban** [comment]
I have it set it up like this connecting to a node in a Pi 4 with Umbrel, however, for some reasons my Spectre Desktop app although connected successfully to the node and showing the funds I don't see those funds on my node wallet.

**Comment by hollisp01** [comment]
https://github.com/cryptoadvance/specter-desktop/discussions/857#discussioncomment-625564
